README.md

```diff
@@ -5,6 +5,8 @@ OmniSciDB is an open source SQL-based, relational, columnar database engine that
 
 To find out more, please check out the [OmniSci Website](https://www.omnisci.com) and the [OmniSciDB wiki](https://github.com/omnisci/omniscidb/wiki/).
 
+For more details see the [OmniSciDB Developer Documentation](https://omnisci.github.io/omniscidb/)
```

Comment:
@alexbaden @dbesemer does the wording here to link to the new sphinx docs look OK?